main.go

```diff
@@ -67,69 +107,44 @@ func main() {
 
 	ctx := context.TODO()
 
-	// Become the leader before proceeding
-	err = leader.Become(ctx, "elasticsearch-operator-lock")
-	if err != nil {
-		ll.Error(err, "failed to become leader")
-		os.Exit(1)
-	}
-
-	// Create a new Cmd to provide shared dependencies and start components
-	mgr, err := manager.New(cfg, manager.Options{
+	mgr, err := ctrl.NewManager(cfg, ctrl.Options{
+		Scheme:             scheme,
 		Namespace:          namespace,
 		MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
+		Port:               9443,
+		LeaderElection:     enableLeaderElection,
+		LeaderElectionID:   "d471c3b1.openshift.io",
```

Comment:
What is the source of this ID?

Reply:
It is auto-generated during `operator-sdk init`. The leader is is hardcoded in the `main.go`[1].

[1] https://v1-1-x.sdk.operatorframework.io/docs/building-operators/golang/advanced-topics/#leader-with-lease